kubetest/objects/networkpolicy.py

```diff
@@ -0,0 +1,106 @@
+"""Kubetest wrapper for the Kubernetes ``NetworkPolicy`` API Object."""
+
+import logging
+
+from kubernetes import client
+
+from .api_object import ApiObject
+
+log = logging.getLogger('kubetest')
+
+
+class NetworkPolicy(ApiObject):
+    """Kubetest wrapper around a Kubernetes `NetworkPolicy`_ API Object.
+
+    The actual ``kubernetes.client.ExtensionsV1beta1Api`` instance that this
+    wraps can be accessed via the ``obj`` instance member.
+
+    This wrapper provides some convenient functionality around the
+    API Object and provides some state management for the `NetworkPolicy`_.
+
+    .. _NetworkPolicy:
+        https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#networkpolicy-v1-networking-k8s-io
+    """
+
+    obj_type = client.NetworkingV1Api
```

Comment:
This should be a reference to the api object class, which I believe is `client.V1NetworkPolicy`